<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvironmentalMonitor</class>
 <widget class="QWidget" name="EnvironmentalMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="home_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/rec/img/home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="light_intensity_lcdNumber">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="temperture_lcdNumber">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="humidity_intensity_lcdNumber">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="gas_intensity_lcdNumber">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
  </widget>
  <widget class="QLabel" name="light_intensity_text_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>光强</string>
   </property>
  </widget>
  <widget class="QLabel" name="light_intensity_icon_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/rec/img/light.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temperture_icon_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/rec/img/temperture.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temperture_text_label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="humidity_icon_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/rec/img/humidity.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="humidity__text_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>湿度</string>
   </property>
  </widget>
  <widget class="QLabel" name="gas_icon_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/rec/img/gas.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gas_text_label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>气体</string>
   </property>
  </widget>
  <widget class="QLabel" name="fan_icon_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="door_icon_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fan_status_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="door_status_label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="light_icon_label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>310</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="light_status_label">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>320</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="body_icon_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="body_status_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>47</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>没人</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
